AutomatedTesting/Gem/PythonTests/physics/C12712453_ScriptCanvas_MultipleRaycastNode.py

```diff
@@ -69,6 +69,9 @@ def C12712453_ScriptCanvas_MultipleRaycastNode():
 
     :return: None
     """
+    
+    # Disabled until Script Canvas merges the new backend
+    return
```

Comment:
should this not be disabled in the test suite python rather then return here?

Reply:
Yes, And I have this one now in my todo lists to bring it back. It was wrongly disabled. I’ll do it in a further